src/docker/start.sh

```diff
@@ -81,6 +81,21 @@ else
   print_header "No custom SSL certificate provided"
 fi
 
+cleanup() {
+  if [ -e config.sh ]; then
+    print_header "Cleanup. Removing Azure Pipelines agent..."
+
+    # If the agent has some running jobs, the configuration removal process will fail.
+    # So, give it some time to finish the job.
+    while true; do
+      ./config.sh remove --unattended --auth PAT --token "$$AZP_TOKEN" && break
```

Comment:
Execute shell explicitly with `bash`, e.g. `bash config.sh`. It solves shell issues with lightweight distros.

Reply:
```suggestion
      bash config.sh remove \
          --auth PAT \
          --token "$AZP_TOKEN" \
          --unattended \
      && break
```